golang/cosmos/x/swingset/keeper/swing_store_exports_handler.go

```diff
@@ -160,7 +160,7 @@ type swingStoreRestoreExportAction struct {
 const (
 	// SwingStoreArtifactModeNone means that no artifacts are part of the
 	// export / import.
-	SwingStoreArtifactModeNone = "none"
+	SwingStoreArtifactModeSkip = "skip"
```

Comment:
I very much would like to not change this as it matches [cosmic-swingset definitions](https://github.com/Agoric/agoric-sdk/blob/4b1403aa06a6c08aae2d4cce0fd582480b7e3c7f/packages/cosmic-swingset/src/export-kernel-db.js#L27)